mage_ai/data_cleaner/statistics/calculator.py

```diff
@@ -209,6 +213,40 @@ def __statistics_overview(self, series, col):
                     dates.sort_values().iloc[math.floor(len(dates) / 2)].isoformat()
                 )
                 data[f'{col}/min'] = dates.min().isoformat()
+            elif column_type == ColumnType.TEXT:
+                text_series = series_non_null
+                num_chars = text_series.apply(lambda string: len(string))
+                data[f'{col}/total_character_count'] = num_chars.sum()
+                data[f'{col}/avg_string_length'] = num_chars.mean()
+                text_series = text_series.str.replace(PUNCTUATION, ' ', regex=True)
+                text_series = text_series.str.lower()
+                text_series = text_series.str.split('\s+')
+                text_series = text_series.apply(
+                    lambda words: [word for word in words if word != '']
+                )
+                word_count = text_series.apply(lambda words: len(words))
+                data[f'{col}/total_word_count'] = word_count.sum()
```

Comment:
Same here. max_word_count and min_word_count are more valuable.

Reply:
Added these two statistics